.pipelines/release-agent.sh

```diff
@@ -4,11 +4,11 @@
 
 # These are plain pipeline variable which can be modified anyone in the team
 # AGENT_RELEASE=cidev
-# AGENT_IMAGE_TAG_SUFFIX=07222021
+# AGENT_IMAGE_TAG_SUFFIX=3.0.0
 
 #Name of the ACR for ciprod & cidev images
 ACR_NAME=containerinsightsprod.azurecr.io
-AGENT_IMAGE_FULL_PATH=${ACR_NAME}/public/azuremonitor/containerinsights/${AGENT_RELEASE}:${AGENT_RELEASE}${AGENT_IMAGE_TAG_SUFFIX}
+AGENT_IMAGE_FULL_PATH=${ACR_NAME}/public/azuremonitor/containerinsights/${AGENT_RELEASE}:${AGENT_IMAGE_TAG_SUFFIX}
 AGENT_IMAGE_TAR_FILE_NAME=agentimage.tar.gz
 
 if [ -z $AGENT_IMAGE_TAG_SUFFIX ]; then
```

Comment:
we should move the error checks to the top i.e. before line 9.

Reply:
moved